fixtures/chauvet-professional/colorado-1-solo.json

```
@@ -6,14 +6,14 @@
   "meta": {
     "authors": ["Da Graca Neves Eric", "Ken Harris"],
     "createDate": "2022-04-19",
-    "lastModifyDate": "2022-04-19"
+    "lastModifyDate": "2023-02-28"
```

A link update is not worth a version increase; the linked information should be roughly the same.

```suggestion
    "lastModifyDate": "2022-04-19"
```

---

No problem. I try to do everything in seperate commits so that it is easy to revert a change like this :)